flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/internal/JdbcTableOutputFormatTest.java

```diff
@@ -66,6 +66,7 @@ public void setup() {
 
 	@Test
 	public void testUpsertFormatCloseBeforeOpen() throws Exception{
+		// FLINK-17544: There should be no NPE thrown from this method
```

Comment:
I think this comment should be belong to `format.close()`.

Reply:
OK. I have modified it.